### PR TITLE
Updated Romanian translation file

### PR DESCRIPTION
**Pull request by mandrill-pie**

### CODE REVIEW

**Comment by erciccione** [comment]
Hi! didn't review it yet, but i see you have some changes in the monero submodule. That should be uncommited since it's pulled directly from the monero repository. can you edit?

**Comment by erciccione** [comment]
Ok i see the submodule is now gone, Can you squash those commits in just one?

**Comment by mandrill-pie** [comment]
@erciccione I'm having trouble understanding why the "monero" submodule still shows up under "changed files". I think I squashed the commits. Let me know if this looks ok

**Comment by erciccione** [comment]
@milargos had the same problem in his PR for the Hebrew language. Probably you also updated  the submodule when pulling upstream. The submodule should be untouched

**Comment by mandrill-pie** [comment]
Ok. But I'm not sure how to move forward. I'm kind of stuck with this submodule...

**Comment by milargos** [comment]
@mandrill-pie seems that your monero submodule is on a previous commit. My issue was that it was on the last commit and it got changed because I did "pull --rebase" instead of just pull. 
Anyway, try going to the monero submodule, run "git pull" and then just reset to the following commit - 8512a83 via "git reset --hard 8512a83"

Not sure that this is the best way to do this but it worked. Then go back the the monero-gui base dir and commit the changes.

**Comment by mandrill-pie** [comment]
Well, `pull` in the monero subfolder didn't work because `Not currently on any branch`.
Ran `git reset --hard 8512a83` and commit, but the submodule is still here.

Did some other silly things after in order ot get rid of the submodule entirely:
```
git submodule deinit -f monero
rm -rf .git/modules/monero
git rm -f monero
```
Now I'm at
```
$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)

	modified:   .gitmodules
	deleted:    monero
```
which doesn't look right to me.
I could use some more help with this.

**Comment by mandrill-pie** [comment]
@erciccione @milargos Ok, finally figured it out! The PR is clean, only 1 commit and only 1 changed file

File: translations/monero-core_ro.ts [hunk 1]
```diff
@@ -1019,27 +1019,27 @@
     <message>
         <location filename="../pages/Settings.qml" line="498"/>
         <source>Debug info</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Informații depanare</translation>
```
Comment (erciccione) [inline translations/monero-core_ro.ts]: If the translation is correct please remove `type="unfinished"`

File: translations/monero-core_ro.ts [hunk 2]
```diff
@@ -1759,34 +1767,35 @@ For the case of outgoing payments, you can get a &apos;Spend Proof&apos; that pr
...
         <source>Message</source>
-        <translation type="unfinished"></translation>
+        <translation>Mesaj</translation>
     </message>
     <message>
         <location filename="../pages/TxKey.qml" line="166"/>
         <location filename="../pages/TxKey.qml" line="278"/>
         <source>Optional message against which the signature is signed</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Mesaj opțional atașat semnăturii</translation>
```
Comment (erciccione) [inline translations/monero-core_ro.ts]: If the translation is correct please remove `type="unfinished"`

Comment (mandrill-pie) [reply translations/monero-core_ro.ts]: **Done**. There are still 14 unfinished: for 11 of them I'm in doubt and 3 of them I cannot translate.

To recap:
- Inside `<name>LeftPanel</name>`:
I don't know what these mean: R, K, B, H, D, M, Y, E S, I

- Inside `<name>AddressBook</name>`, this one seems dubious in English:
`Payment ID. A unique user name used in the address book. It is not a transfer of information sent during the transfer.`

- And I was unable to translate:
`Successfully rescanned spent outputs`
`Seep unmixable`
`No unmixable outputs to sweep`

**Comment by mandrill-pie** [comment]
@erciccione My latest comment got lost somehow... Anyway, I made all the necessary changes.

File: translations/monero-core_ro.ts [hunk 3]
```diff
@@ -454,17 +454,17 @@
...
         <location filename="../LeftPanel.qml" line="439"/>
         <source>Prove/check</source>
-        <translation type="unfinished"></translation>
+        <translation>Dovadă/Verificare</translation>
     </message>
     <message>
         <location filename="../LeftPanel.qml" line="440"/>
         <source>K</source>
-        <translation>K</translation>
+        <translation type="unfinished">K</translation>
```
Comment (jonathancross) [inline translations/monero-core_ro.ts]: Looks "finished" to me, correct?
Please check other letters below: B H D M Y E S I - somehow these all got marked as "unfinished".

Comment (jonathancross) [reply translations/monero-core_ro.ts]: Other than that, looks great. :-)

Comment (erciccione) [reply translations/monero-core_ro.ts]: @mandrill-pie take a look at the discussion here about shortcuts: https://github.com/monero-project/monero-gui/pull/1133#discussion_r170714196

**Pull request merged**